### ISSUE TITLE
Popover – tabbing in reverse traps focus

### ISSUE THREAD

**Issue by nahumzs** [acl-services/paprika#33]
# Bug Report

*Please fill in as much detail as possible below*

## Expected behavior
The popover should seamlessly navigate throw all interactive element via `tab` and `shift+tab`. 

## Current behavior
The popover has a wrong behaviour when using `shift+tab` on popover when the `isEager` Prop set to true (`Tooltips`). 

Focus is robbed from the `shift-tab` making it impossible to navigate to the element before the popover.

From an accessibility perspective, we lack of the onFocus event, which according to the A11y linter we should use.
 
## Screenshots / Gifs / Codepens
this can be seen in storbyook in most of the *Popover* stories

cc @mikrotron

**Comment by jamiek-acl** [comment]
Not seeing this behaviour (also tried on the `OverflowMenu`), must have been solved in the past.

![tabbing](https://user-images.githubusercontent.com/23224777/116451954-01eb2f00-a812-11eb-90f6-e783497d8f6b.gif)

Closing issue.
